File: CS/WindowsApplication3/ViewModels/OrderViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DXSample.Common.ViewModels;
using DXSample.Models;


namespace DXSample.ViewModels {
    public class OrderViewModel : BaseViewModel<Order> {
        protected override void LoadEntities() {
            base.LoadEntities();
            for (int i = 0; i < 10; i++) {
                Entities.Add(new Order() { ID = Entities.Count, OrderDate = DateTime.Now.AddDays(-i), ProductType = i % 4 });
            }
        }
    }
}

File: VB/WindowsApplication3/ViewModels/OrderViewModel.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DXSample.Common.ViewModels
Imports DXSample.Models


Namespace DXSample.ViewModels
    Public Class OrderViewModel
        Inherits BaseViewModel(Of Order)

        Protected Overrides Sub LoadEntities()
            MyBase.LoadEntities()
            For i As Integer = 0 To 9
                Entities.Add(New Order() With {.ID = Entities.Count, .OrderDate = Date.Now.AddDays(-i), .ProductType = i Mod 4})
            Next i
        End Sub
    End Class
End Namespace
